src/v2/Apps/Partner/Components/PartnerHeader/PartnerHeaderImage.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { useEffect, useState } from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { PartnerHeaderImage_profile } from "v2/__generated__/PartnerHeaderImage_profile.graphql"
+import { FullBleed } from "v2/Components/FullBleed"
+import styled from "styled-components"
+
+const Container = styled(FullBleed)`
+  position: relative;
+  overflow: hidden;
+  background-attachment: fixed;
+`
+
+const Image = styled.img<{ scrollPosition: number }>`
```

Comment:
For anything with a large number of values like this just use `style={{ }}` which will avoid styled-components from generating a large amount of classes.